packages/web/src/components/HomeLink.tsx

```diff
@@ -1,29 +1,20 @@
-import { THomeButtonAnchor, THomeButtonRouterLink } from "../webTypes";
-
 import { Link } from "react-router-dom";
 import React from "react";
+import { THomeButtonRouterLink } from "../webTypes";
 
 interface HomeLinkPropsBase {
   children: React.ReactElement | React.ReactElement[];
 }
 
-type HomeRouterLinkProps = HomeLinkPropsBase & THomeButtonRouterLink;
-type HomeAnchorProps = HomeLinkPropsBase &
-  THomeButtonAnchor &
-  React.HTMLProps<HTMLAnchorElement>;
+type HomeLink = HomeLinkPropsBase & THomeButtonRouterLink;
 
-export const HomeRouterLink = (props: HomeRouterLinkProps) => {
-  const { children, linkProps, ...rest } = props;
+const HomeLink = (props: HomeLink) => {
+  const { children, linkProps } = props;
   return (
-    <Link {...linkProps} {...rest}>
+    <Link style={{ textDecoration: "none" }} {...linkProps}>
```

Comment:
This update was just to get rid of some console warnings shown on the home page where improper props were being added to the DOM element.

Reply:
I like it!